test/start-kubernetes.sh

```diff
@@ -17,15 +17,14 @@ LOCKFILE="${LOCKFILE:-${REPO_DIRECTORY}/_work/start-kubernetes.exclusivelock}"
 LOCKDELAY="${LOCKDELAY:-300}" # seconds
 NODES=( $DEPLOYMENT_ID-master
         $DEPLOYMENT_ID-worker1
-        $DEPLOYMENT_ID-worker2
-        $DEPLOYMENT_ID-worker3)
+        $DEPLOYMENT_ID-worker2)
 CLOUD="${CLOUD:-true}"
 FLAVOR="${FLAVOR:-medium}"
 SSH_KEY="${SSH_KEY:-${RESOURCES_DIRECTORY}/id_rsa}"
 SSH_PUBLIC_KEY="${SSH_KEY}.pub"
 EFI="${EFI:-true}"
 KVM_CPU_OPTS="${KVM_CPU_OPTS:-\
- -m 2G,slots=${TEST_MEM_SLOTS:-2},maxmem=34G -smp 4\
+ -m 2G,slots=${TEST_MEM_SLOTS:-2},maxmem=34G -smp 2\
```

Comment:
I think this can be managed in CI environment configuration.

Reply:
I was considering that, but then asked myself whether it's actually useful to then have a larger cluster on a desktop. My answer was "no", therefore I'm changing the defaults here.